### ISSUE TITLE
MMO Instance Memory Spike Up Each Instance

### ISSUE THREAD

**Issue by tubhluas** [suriyun-production/mmorpg-kit-docs#2175]
https://pasteboard.co/cFRlIml2ah9C.jpg

Memory keep increasing on ubuntu server. Couldn't find any error or logs about it. This happened when enter instance. It looks like memory are not release after loading and it will keep increasing each time until memory hit 100% and server crashed.

btw, is memory on server suppose to increase every second or is this a memory leaking?

**Comment by tubhluas** [comment]
So, i still trying to figure out something is probably memory leaking on a script somewhere. I did replaced that 2 lines and haven't test it with instance yet. however the memory is still increasing bit by bit every second/minute. I did a drop cache command in ubuntu however used memory won't go down. Let me know if you found anything on it. I will repost here if I found something.

![Memory Spike](https://github.com/suriyun-production/mmorpg-kit-docs/assets/133326174/7dee537d-ad56-4c9b-923b-4e2426d04080)
![Memory Increasing2](https://github.com/suriyun-production/mmorpg-kit-docs/assets/133326174/10a12169-9489-4b70-a947-4cc0b8c9f6cf)
